src/components/Trackpad/FloatingMirror.tsx

```
@@ -0,0 +1,253 @@
+                )}
+            </div>
+
+            {/* Top bar (close + fps) */}
+            <div className="absolute top-0 left-0 right-0 flex items-center justify-between px-2 py-1 bg-gradient-to-b from-black/70 to-transparent pointer-events-none">
+                <span className="text-[9px] font-mono text-white/50">
+                    {stalled ? '⚠ Stalled' : fps > 0 ? `${fps} fps` : '…'}
+                </span>
+                <button
+                    className="pointer-events-auto w-4 h-4 rounded-full bg-red-500/80 hover:bg-red-400 flex items-center justify-center text-[8px] text-white leading-none"
+                    onPointerDown={(e) => { e.stopPropagation(); onClose(); }}
+                >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)